examples/userdemo/connectionstyle_demo.py

```diff
@@ -30,7 +30,7 @@ def demo_con_style(ax, connectionstyle):
             transform=ax.transAxes, ha="left", va="top")
 
 
-fig, axs = plt.subplots(3, 5, figsize=(8, 4.8))
+fig, axs = plt.subplots(3, 5, figsize=(7, 4), constrained_layout=True)
```

Comment:
https://61514-1385122-gh.circle-artifacts.com/0/doc/build/html/gallery/userdemo/connectionstyle_demo.html#sphx-glr-gallery-userdemo-connectionstyle-demo-py

With the reduced size, the text becomes very dominating and visually too close to the arrows:
![grafik](https://user-images.githubusercontent.com/2836374/124660566-a9d24800-dea6-11eb-8918-60db25d27cb3.png)
![grafik](https://user-images.githubusercontent.com/2836374/124660615-bb1b5480-dea6-11eb-9558-0e10dd2c5307.png)

This needs more fine-tuning, i.e.  moving points and/or changing font size and/or maxing out the available figure size.

I propose to hold off such difficult fine-tuning until we know what size will finally be available (xref https://github.com/pydata/pydata-sphinx-theme/issues/427, https://github.com/pydata/pydata-sphinx-theme/pull/428).

Reply:
I agree with @timhoffm  on both accounts.

Reply:
This should be better now:
![image](https://user-images.githubusercontent.com/302469/126023290-c394fa2b-191e-40ad-9576-f6cce0faffcd.png)